requirements.txt

```diff
@@ -5,4 +5,4 @@ numpy>=1.16.4
 torch>=1.1
 tensorboard>=1.14
 future>=0.17.1  # required for builtins in setup.py
-
+pre-commit>=1.21.0
```

Comment:
Isn't this more like a test requirement?

Reply:
It is more of a development requirement, because I add it in the pre-commit form. So every time you commit changes it will run black formater.

Reply:
But probably it should be somewhere else

Reply:
this belongs to tests/requirements.txt